### PR TITLE
feat(admission-controller): Deprecate sysdig.url (keep for compatibility) and use apiEndpoint instead

### PR DESCRIPTION
**Pull request by airadier**
## What this PR does / why we need it:

As discussed internally, the AC chart  uses sysdig.url instead of sysdig.apiEndpoint and that expects the URL to have an `https://` prefix.

With this change, the AC chart aligns better with the existing subcharts.

Backwards compatibility is kept by still supporting sysdig.url

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/admission-controller/templates/webhook/configmap.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ metadata:
   labels:
     {{ include "admissionController.webhook.labels" . | nindent 4 }}
 data:
-  SECURE_URL: "{{ include "admissionController.apiEndpoint" . }}"
+  SECURE_URL: "{{ .Values.sysdig.url | default (printf "https://%s" (include "admissionController.apiEndpoint" .)) }}"
```
Comment (mavimo) [inline charts/admission-controller/templates/webhook/configmap.yaml:9]: Are we sure is always HTTPS?

File: charts/admission-controller/templates/scanner/configmap.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@ metadata:
   labels:
     {{ include "admissionController.scanner.labels" . | nindent 4 }}
 data:
-  SECURE_BASE_URL: "{{ include "admissionController.apiEndpoint" . }}"
+  SECURE_BASE_URL: "{{ .Values.sysdig.url | default (printf "https://%s" (include "admissionController.apiEndpoint" .)) }}"
```
Comment (mavimo) [inline charts/admission-controller/templates/scanner/configmap.yaml:10]: Are we sure is always HTTPS?

Comment (airadier) [reply charts/admission-controller/templates/scanner/configmap.yaml:10]: It should be HTTPS, at least for production. It is a strong assumption, but we are making the same assumption in other places like:

- https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/templates/configmap-host-analyzer.yaml#L12
- https://github.com/sysdiglabs/charts/blob/master/charts/node-analyzer/templates/configmap-host-scanner.yaml#L10
- https://github.com/sysdiglabs/charts/blob/master/charts/rapid-response/templates/configmap.yaml#L8

**Review by dimar-apolicy — COMMENTED** [review]
@airadier no objections from my side, as long as it approved by @lior-alafi-work we good to go

**Review by lior-alafi-work — APPROVED** [review]
cool once merged i'll pull to the kspm ac branch so we can be aligned with that value

**Pull request merged**